src/index.js

```diff
@@ -69,7 +69,15 @@ function preInit(initPromise) {
     dataView.preInit(initPromise);
 }
 
-function init(promise, resolve, reject) {
+function init() {
+    // setup init promise
+    let resolve;
+    let reject;
```

Comment:
None of the `postRender` functions actually use the `promise` so this can be refactored/simplified.